client/src/components/browse-recipes/Browse.svelte

```diff
@@ -0,0 +1,39 @@
+<script>
+  import { onMount } from 'svelte';
+  import { slide } from 'svelte/transition';
+  import Header from '../header/Header.svelte';
+  import Navigation from '../navigation/Navigation.svelte';
+  import Category from './Category.svelte';
+  import Recipes from './Recipes.svelte';
+  import recipesApi from '@/api/recipesApi';
```

Comment:
this should be exported from `src/api/index` and then you can do
```js
import { recipesApi } from '@/api';
```
which is cleaner

If you need help just look at one of my currently active PRs